src/main/java/com/gregtechceu/gtceu/api/item/MetaMachineItem.java

```diff
@@ -62,4 +55,15 @@ protected boolean placeBlock(BlockPlaceContext context, BlockState state) {
         }
         return superVal;
     }
+
+    @Override
+    public void initializeClient(Consumer<IClientItemExtensions> consumer) {
+        consumer.accept(new IClientItemExtensions() {
```

Comment:
This should be a separate static class, idk if java is smart enough to realise this here